_data/integrations.yml

```diff
@@ -0,0 +1,626 @@
+---
+#- id: [lowercase, words separated by hyphen]
+#  name:
+#  url:
+#  type: [ tms, cat ]
+#  platform: [ cloud-based or desktop ]
```

Comment:
What if it's on both desktop, mobile and cloud?  Maybe `platforms` (plural)?

That said, I don't think it's our business to track this.